google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/GCRules.java

```diff
@@ -30,7 +30,11 @@
 
 // TODO(igorbernstein2): the distinction between GcRule & GCRule is too subtle, use fully qualified
 // names for the protos.
-/** Wraps {@link GcRule} protocol buffer object and exposes a simpler Fluent DSL model */
+/**
+ * Wraps {@link GcRule} protocol buffer object and exposes a simpler Fluent DSL model
+ *
+ * <p>Note: GCRule are only applied when compaction starts for the table.
```

Comment:
Note: Garbage collection executes opportunistically in the background, and
so it's possible for reads to return a cell even if it matches the active
GC expression for its family.  Please use appropriate filters in your reads to exclude
cells which are eligible for garbage collection